launcher/modplatform/flame/FlameModIndex.cpp

```diff
@@ -51,7 +41,7 @@ void FlameMod::loadIndexedPackVersions(ModPlatform::IndexedPack& pack,
     for (auto versionIter : arr) {
         auto obj = versionIter.toObject();
 
-        auto versionArray = Json::requireArray(obj, "gameVersion");
+        auto versionArray = Json::requireArray(obj, "gameVersions");
```

Comment:
maybe they did something with the way versions are represented in the strings, so the validation in `FlameModPage::validadeVersion` is not working properly...

Reply:
CurseForge, nothing we can do